integration-tests/elasticsearch-java-client/src/main/resources/META-INF/resources/fruits.html

```diff
@@ -0,0 +1,120 @@
+<!doctype html>
```

Comment:
This file is not necessary for an integration test... ?
Maybe you should move this to a quickstart?

Reply:
Yes, those are copied from the High Level client IT that is the same as the quickstart and include them.
I'll remove them.

Reply:
removed